### PR TITLE
Make heading optional for Google Doc copy

### PR DESCRIPTION
**Pull request by jranks123**
## What does this change?
We had some issues with the Google Doc where the heading was left blank, which was causing an error in frontend. Heading is optional anyway, so we shouldn't throw an error if the heading is left blank

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js [hunk 1]
```diff
@@ -568,10 +567,7 @@ export const getEpicTestsFromGoogleDoc = (): Promise<
                                 ','
                             ),
                             copy: {
-                                heading: throwIfEmptyString(
-                                    'heading',
-                                    row.heading
-                                ),
+                                heading: row.heading,
```
Comment (gtrufitt) [inline static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js]: In [the template](https://github.com/guardian/frontend/blob/9a414904ba2bb13fdd7707ac3d26bd11f7b12092/static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-control.js#L27) you provide a default in the template of empty string but don't remove the `<h2>` so you end up with an empty element.

**Comment by gtrufitt** [comment]
Can this 🚢 or 💀?

**Review by sndrs — COMMENTED** [review]
shall we close this?

**Comment by gtrufitt** [comment]
I am going to close this PR for now

**Pull request closed without merging**